plugins/lsp/language-servers/default.nix

```
@@ -212,6 +212,12 @@ let
         "graphql-language-service-cli"
       ];
     }
+    {
+      name = "harper-ls";
+      description = "The Grammar Checker for Developers";
+      serverName = "harper_ls";
+      package = pkgs.harper;
```

This was recently changed, we now expect the form:
```suggestion
      package = "harper";
```

This is evaluated in the form `getAttrFromPath (toList package) pkgs`, and is used to allow showing a user-friendly `defaultText` in the docs, i.e. `"pkgs.${concatStringSep "." (toList package)}"`